codingTest_김준우/두 정수 사이의 합.js

```diff
@@ -0,0 +1,24 @@
+function solution(a, b) {
+  let answer = 0;
+  if (a === b) {
+    answer = a || b;
+  } 
+  else {
+    if (a > b)
+    {
+      for (let i = b; i <= a; i++)
+      {
+        answer += i;
+      }
+    } 
+    else 
+    {
+      for (let i = a; i <= b; i++) 
+      {
+        answer += i;
+      }
+    }
```

Comment:
```suggestion
  else if (a > b)  {
      for (let i = b; i <= a; i++) {
        answer += i;
      }
  } else {
      for (let i = a; i <= b; i++) {
        answer += i;
      }
  }
```

else안에 if 문을 또 만들기 보단 else if ()를 사용하면 될 것 같아요.